examples/DotnetPush/DotnetPush.Android/Properties/AssemblyInfo.cs

```diff
@@ -6,14 +6,9 @@
 // General Information about an assembly is controlled through the following
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
-[assembly: AssemblyTitle("DotnetPush.Android")]
-[assembly: AssemblyDescription("")]
-[assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("")]
-[assembly: AssemblyProduct("DotnetPush.Android")]
-[assembly: AssemblyCopyright("Copyright ©  2021")]
-[assembly: AssemblyTrademark("")]
-[assembly: AssemblyCulture("")]
+[assembly: AssemblyCompany("Ably Realtime")]
```

Comment:
These days, outside of context of referring to full legal entity name, we seem to be dropping the 'Realtime' suffix:

```suggestion
[assembly: AssemblyCompany("Ably")]
```